index.ts

```diff
@@ -21,29 +21,28 @@ export function* sinergia(
       const itemIterator: IterableIterator<any> = task(accumulator, item);
 
       yield new Promise(resolve => {
-        const step = (timestamp) => { // timestamp not used
+        const step = () => {
           const iteration = itemIterator.next();
 
-          if (!iteration.done) {
-            window.requestAnimationFrame(step);
-          }
-          else {
+          if (iteration.done) {
```

Comment:
I'm well aware of this rule, but in this case you can't say it's difficult to understand :D Also there isn't any double negation.
I think that `stiamo smacchiando i giaguari` here. ESLint is about code-style, not bloody rules.